http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/OpenAPIIT.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.ts.http.minimum;
+
+import static io.restassured.RestAssured.given;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@QuarkusScenario
+public class OpenAPIIT {
+    @Test
+    public void smoke() {
+        given().get("/api/q/openapi").then().statusCode(HttpStatus.SC_OK);
+    }
+
+    @Test
+    public void validation() throws JsonProcessingException {
```

Comment:
- I would add a tag pointing to the issue that is behind this test.
- Try to find a better tets name: 
Suggestion
- `validation` -> `openapiSizeAndPatternConstraints`

Reply:
renamed the method